.github/workflows/secrel.yml

```
@@ -9,7 +9,8 @@ on:
   # Run on a schedule, at 1100 UTC/0700 EST every weekday MON-FRI
```

```suggestion
  # Run on a schedule, at 1100 UTC/0700 EST every weekday MON-FRI
  # (for debugging: Run on a schedule, once an hour on the 30th minute every weekday MON-FRI)
```
minor suggestion, just to acknowledge the comment associated with the cron string